tests/baselines/reference/jsdocParameterTagSnippetCompletion2.baseline

```diff
@@ -0,0 +1,3608 @@
+=== /tests/cases/fourslash/b.js ===
+// /**
+//  * 
+//    ^
+// | ----------------------------------------------------------------------
+// | @abstract
+// | @access
+// | @alias
+// | @argument
+// | @async
+// | @augments
+// | @author
+// | @borrows
+// | @callback
+// | @class
+// | @classdesc
+// | @constant
+// | @constructor
+// | @constructs
+// | @copyright
+// | @default
+// | @deprecated
+// | @description
+// | @emits
+// | @enum
+// | @event
+// | @example
+// | @exports
+// | @extends
+// | @external
+// | @field
+// | @file
+// | @fileoverview
+// | @fires
+// | @function
+// | @generator
+// | @global
+// | @hideconstructor
+// | @host
+// | @ignore
+// | @implements
+// | @inheritdoc
+// | @inner
+// | @instance
+// | @interface
+// | @kind
+// | @lends
+// | @license
+// | @link
+// | @linkcode
+// | @linkplain
+// | @listens
+// | @member
+// | @memberof
+// | @method
+// | @mixes
+// | @module
+// | @name
+// | @namespace
+// | @overload
+// | @override
+// | @package
+// | @param
+// | @private
+// | @prop
+// | @property
+// | @protected
+// | @public
+// | @readonly
+// | @requires
+// | @returns
+// | @satisfies
+// | @see
+// | @since
+// | @static
+// | @summary
+// | @template
+// | @this
+// | @throws
+// | @todo
+// | @tutorial
+// | @type
+// | @typedef
+// | @var
+// | @variation
+// | @version
+// | @virtual
+// | @yields
+// | @param {*} b 
```

Comment:
couple of display questions:

1. Do these longer completions sort below all the other tag names in a real editor? It might not be important if the prefix is `@pa` and there are only 3 anyway.
2. In editors that create a jsdoc template upon typing `/**`, what opportunities are there to show this completion? Do we also need to improve that template with this feature?

Reply:
They have the same `sortText` as the keywords, which is the only aspect we control on our side on sorting, so it's up to vscode to sort them first or not. I think they show up later in the baselines because they are last in the array of completion entries. I thought using the same `sortText` would be ok, since (1) we're also returning all of the possible jsdoc tags, so we're not sure which one you want, and (2) vscode is going to filter that list as you type, so typing `@p` already reduces the list of entries to something that fits in the display without needing to scroll.

Reply:
Re 2, if we want we could change the jsdoc comment template completion to reuse this code for destructuring in JS (and in TS depending on what we decide to do).

Reply:
I vote to make 2 happen at some point; the new code is better than that the template currently has.